docs/src/main/sphinx/connector/hudi.md

```diff
@@ -82,6 +82,11 @@ Additionally, following configuration properties can be set depending on the use
   - Maximum number of metastore data objects per transaction in the Hive
     metastore cache.
   - `2000`
+* - `hudi.query-partition-filter-required`
+  - Set to `true` to force a query to use a partition filter. You can use the
```

Comment:
Do we have any docs on what a partition filter is that we can link to?

Reply:
We do not have any doc. `partition filter` here refers to partition column used in filter condition.

Set to `true` to  force a query to use a partition column in the filter condition.

Reply:
@mosabua, Thanks for the review. I have added some more content explaining the behavior of this property.

Reply:
Commented.. thanks!